src/projector.ts

```diff
@@ -29,9 +29,17 @@ export interface ProjectorOptions extends ParentMixinOptions<Child>, EventedOpti
 	 *
 	 * Only applies if `autoAttach` is `true`
 	 */
+	// FIXME Is this used? What about replace?
```

Comment:
Yes it is used in the initializer... it gets passed to `attach`, though potentially this should be changed to be `AttachOptions` now.